### PR TITLE
Add shellcheck violation tool in diff-quality

### PR DESCRIPTION
**Pull request by mg-castel**

### CODE REVIEW

File: diff_cover/violationsreporters/violations_reporter.py [hunk 1]
```diff
@@ -474,6 +474,17 @@ def measured_lines(self, src_path):
     command_to_check_install=["jshint", "-v"],
 )
 
+shellcheck_driver = RegexBasedDriver(
+    name="shellcheck",
+    supported_extensions=["sh"],
+    # User gcc format to ease violations parsing
```
Comment (Bachmann1234) [inline diff_cover/violationsreporters/violations_reporter.py:480]: nitpick "use", right?

**Comment by Bachmann1234** [comment]
This is looking good, Found a typo in the comments and the build is failing cuse you need to format the code with `black`. But I can take another look after that. 

Thanks for the PR!

File: diff_cover/violationsreporters/violations_reporter.py (hunk 1 above)
Comment (mg-castel) [reply diff_cover/violationsreporters/violations_reporter.py]: Yep, thx

**Comment by Bachmann1234** [comment]
Alright, lets get this merged! Thanks for the PR!

**Comment by Bachmann1234** [comment]
@mg-castel released https://pypi.org/project/diff-cover/7.6.0/!

**Pull request merged**